pkg/kubectl/cmd/logs.go

```diff
@@ -56,7 +56,7 @@ var (
 		kubectl logs -f -c ruby web-1
 
 		# Display only the most recent 20 lines of output in pod nginx
-		kubectl logs --tail=20 nginx
+		kubectl logs --tail=30 nginx
```

Comment:
This is only an example. I don't think we need this change.
BTW, the comment is still not updated.